code/game/objects/items/weapons/defib.dm

```diff
@@ -105,16 +105,13 @@
 	return
 
 /obj/item/weapon/melee/defibrillator/proc/shockAttack(mob/living/carbon/human/target,mob/user)
+	var/damage = rand(30, 60)
+	if (!target.electrocute_act(damage, src, def_zone = LIMB_CHEST))
+		return
 	var/datum/organ/internal/heart/heart = target.get_heart()
 	if(heart)
 		heart.damage += rand(5,60)
-	target.visible_message("<span class='danger'>[target] has been shocked in the chest with the [src] by [user]!</span>")
-	var/incapacitation_duration = rand(6, 12)
-	target.Knockdown(incapacitation_duration)
-	target.Stun(incapacitation_duration)
```

Comment:
Why was all this discarded?

Reply:
It's now being handled by electrocute_act() (that's where the slime flag check lives)